R/add_phylopic.r

```diff
@@ -5,23 +5,36 @@
 #'
 #' @param img A [Picture][grImport2::Picture-class] or png array object, e.g.,
 #'   from using [get_phylopic()].
-#' @param name A taxonomic name to be passed to [get_uuid()].
-#' @param uuid A valid uuid for a PhyloPic silhouette (such as that returned by
-#'   [get_uuid()] or [pick_phylopic()]).
-#' @param x x value of the silhouette center. Ignored if `y` and `ysize` are not
-#'   specified.
-#' @param y y value of the silhouette center. Ignored if `x` and `ysize` are not
-#'   specified.
-#' @param ysize Height of the silhouette. The width is determined by the aspect
-#'   ratio of the original image. Ignored if `x` and `y` are not specified.
-#' @param alpha A value between 0 and 1, specifying the opacity of the
-#'   silhouette (0 is fully transparent, 1 is fully opaque).
-#' @param color Color to plot the silhouette in.
+#' @param name \code{character}. A taxonomic name to be passed to [get_uuid()].
+#' @param uuid \code{character}. A valid uuid for a PhyloPic silhouette (such as
+#'   that returned by [get_uuid()] or [pick_phylopic()]).
+#' @param x \code{numeric}. x value of the silhouette center. Ignored if `y` and
+#'   `ysize` are not specified.
+#' @param y \code{numeric}. y value of the silhouette center. Ignored if `x` and
+#'   `ysize` are not specified.
+#' @param ysize \code{numeric}. Height of the silhouette. The width is
+#'   determined by the aspect ratio of the original image. Ignored if `x` and
+#'   `y` are not specified.
+#' @param alpha \code{numeric}. A value between 0 and 1, specifying the opacity
+#'   of the silhouette (0 is fully transparent, 1 is fully opaque).
+#' @param color \code{character}. Color to plot the silhouette in.
+#' @param horizontal \code{logical}. Should the silhouette be flipped
+#'   horizontally?
+#' @param vertical \code{logical}. Should the silhouette be flipped vertically?
+#' @param angle \code{numeric}. The number of degrees to rotate the silhouette
```

Comment:
```suggestion
#' @param angle \code{numeric}. The number of degrees to rotate the silhouette by
```
Needed?

Reply:
If you want to be grammatically correct with "by", you'd need to say "The number of degrees by which to rotate the silhouette", and that just sounds like too much to me.